examples/active_fire_detection/README.md

```diff
@@ -0,0 +1,205 @@
+
+# Preparing Satellite Data for Wildfire Detection with Oxen.ai 🔥🛰️
+### Introduction
+There are few domains with a greater quantity of high-quality, publicly accessible data than Earth-observing satellite imagery. Government-funded missions such as  [LANDSAT](https://www.usgs.gov/landsat-missions#:~:text=Since%201972%2C%20Landsat%20satellites%20have,natural%20resources%20and%20the%20environment.) (USGS / NASA) and [SENTINEL](https://sentinel.esa.int/web/sentinel/missions/sentinel-2) (ESA) have captured and transmitted terabytes of this data every day for multiple decades, providing an extremely rich resource to those wishing to develop models for pressing geospatial challenges like disaster response. 
+
+Researchers have built and meticulously annotated domain-specific datasets from this vast array of satellite imagery, such as [this dataset](https://drive.google.com/drive/folders/1GIcAev09Ye4hXsSu0Cjo5a6BfL3DpSBm) for space-based active fire detection introduced in [this paper by Pereira et. al](https://arxiv.org/abs/2101.03409).
+
+While these datasets are information-rich and freely available, they are notoriously difficult to collaborate on for machine learning applications due to a few key factors:
```

Comment:
I'm open to this being too much for now and better saved for a later "why satellite data is hard" style post